keyboards/gray_studio/aero75/keymaps/vial/config.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
Unfortunately PRs to QMK main will not be accepted with any vial elements, these must be removed

Reply:
@keyboard-magpie Thank you for informing me, I have removed the vial part